src/style.css

```diff
@@ -1 +1,85 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
In the `font-face` rule, the `font-family` property should not include fallback fonts. It should only include the name of the font you're importing. The fallback fonts should be specified where you use the font. (CSS issue)